### ISSUE TITLE
Grafana 3.1.1 : Cannot find influxdb series name

### ISSUE THREAD

**Issue by uzfarid** [grafana/grafana#5966]
* **I'm submitting a ...**
- [x ] Bug report
- [ ] Feature request
- [ ] Question / Support request: **Please do not** open a github issue. [Support Options](http://grafana.org/support/)

Please include this information:
- What Grafana version are you using?
3.1.1
- What datasource are you using?
Influxdb 0.8
- What OS are you running grafana on?
Amazon linux
- What did you do?
Tried to add new query
- What was the expected result?
Expecation was grafana could display all metrics available starting by that string name given.
- What happened instead?
It now seems require to give exact series name. In very early versions of grafana, basically when we types "jmeter." then it was able to find all metrics available starting with that series name.

Is this how it works with latest versions of grafana or am I doing something wrong during adding a query? 

Basically i click add query and give beginning of jmeter series name "like jmeter." , it says it cant find that series instead of displaying me all metrics available starting with that name.

**Comment by torkelo** [comment]
maybe something wrong with your InfluxDB server? check in chrome dev tools what network requests are being issued and the response from InfluxDB 

please do not open issues relating to InfluxDB 0.8 in this repo: Use the plugin repo (https://github.com/grafana/influxdb-08-datasource) 

There is no support for that old plugin btw,
